target/linux/mediatek/dts/mt7981b-snr-snr-cpe-ax2.dts

```diff
@@ -0,0 +1,348 @@
+// SPDX-License-Identifier: GPL-2.0-only OR MIT
+
+/dts-v1/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+#include "mt7981.dtsi"
+
+/ {
+	model = "SNR SNR-CPE-AX2";
+	compatible = "snr,snr-cpe-ax2", "mediatek,mt7981";
+
+	aliases {
+		label-mac-device = &gmac0;
+
+		led-boot = &led_gear_blue;
+		led-failsafe = &led_gear_red;

```

Comment:
Did you make tests (compile and run) after the changes? You also deleted this necessary labels `led_gear_blue`, `led_gear_red` from the leds section.